### PR TITLE
MS Graph - filter fix

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23532

## Description
Fixing the way filters are implemented in MS Graph

### CODE REVIEW

**Comment by ronykoz** [comment]
@amshamah419 please go over RN

File: Packs/Legacy/Integrations/MicrosoftGraphSecurity/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Fixed an issue where filters were not properly implemented in ***msg-search-alerts*** command.
```
Comment (amshamah419) [inline Packs/Legacy/Integrations/MicrosoftGraphSecurity/CHANGELOG.md:2]:
```suggestion
Fixed an issue where filters were not properly implemented in the ***msg-search-alerts*** command.
```

**Comment by roysagi** [comment]
removing docs approved - @kirbles19 if you can look at the note in the changelog please

**Comment by kirbles19** [comment]
@roysagi Done

**Pull request merged**